presto-docs/src/main/sphinx/plugin/redis-hbo-provider.rst

```diff
@@ -0,0 +1,61 @@
+=======================
+Redis Hbo Provider
```

Comment:
```suggestion
Redis HBO Provider
```

I would keep capitalization the same in the docs, all caps because it's an acronym.

Reply:
The acronym HBO occurs in more places than this line - line 5 and line 11 as of this comment and perhaps elsewhere - so I would suggest changing this throughout the page for consistency.